libs/common/src/main/kotlin/person/Foedselsnummer.kt

```diff
@@ -90,7 +98,13 @@ class Foedselsnummer private constructor(@JsonValue val value: String) {
      *
      * Brukes også for identer i test som er opprettet som "NAV syntetisk" i Dolly
      */
-    private fun isHNumber(): Boolean = Character.getNumericValue(value[2]) >= 4
+    private fun isHNumber(): Boolean = Character.getNumericValue(value[2]) in 4..7
```

Comment:
om man plusser på 40 på måned så skulle jeg tro vi havnet på 41-52, så når skal måned starte med for eksempel 6. Netoden sier jo nå at om måneden er 63 så er det et H-nummer. Er sikkert ikke av så stor konsekvens, men litt rart